src/components/panes/SmsDistributionCreditTransactionPane.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { injectIntl, FormattedMessage as Msg } from 'react-intl';
+import { connect } from 'react-redux';
+
+import PaneBase from './PaneBase';
+
+import InfoList from '../misc/InfoList';
+import LoadingIndicator from '../misc/LoadingIndicator';
+
+import {
+    retrieveSmsDistributionCreditTransaction,
+} from '../../actions/smsDistribution';
+import { replacePane } from '../../actions/view';
+import { getListItemById } from '../../utils/store';
+
+const oreToKrona = (ore = 0) => (ore / 100).toFixed(2);
+
+const cn = (suffix = '') => `SmsDistributionCreditTransactionPane${suffix}`;
+const msgId = suffix => `panes.smsDistributionCreditTransaction.${suffix}`;
+
+const mapStateToProps = (state, { paneData: { params: [transactionId] } }) => ({
+    transactionItem: getListItemById(
+        state.smsDistributions.transactionList,
+        transactionId,
+    ),
+});
+
+@connect(mapStateToProps)
+@injectIntl
+export default class SmsDistributionCreditTransactionPane extends PaneBase {
+    componentDidMount() {
+        super.componentDidMount();
+
+        const transactionId = this.getParam(0);
+
+        this.props.dispatch(retrieveSmsDistributionCreditTransaction(transactionId));
+    }
+
+    componentDidUpdate() {
+        const {
+            type,
+            distributionId,
+        } = this.getRenderData();
+
+        if (type === 'distribution') {
+            this.props.onReplace('smsdistribution', [distributionId]);
+        }
+    }
+
+    getRenderData() {
+        const {
+            intl: {
+                formatMessage,
+            },
+            transactionItem,
+        } = this.props;
+
+        const isLoaded = !!(transactionItem && !transactionItem.isPending && transactionItem.data);
+
+        if (!isLoaded) {
+            return {
+                formatMessage,
+
+                isLoaded,
+            };
+        }
+
+        const {
+            id,
+            type,
+            created,
+            amount,
+
+            distribution_id: distributionId,
+
+            description,
+
+            charge: {
+                amount: chargeAmount,
+                last4
+            } = {},
+
+        } = transactionItem.data;
+
+        const receiptUrl = 'http://example.com#not-implemented';
```

Comment:
I can't figure out how to implement this. The core endpoint is `/orgs/org_id:/sms_distribution_credit_transactions/:transaction_id/receipt`, which returns a PDF.